_data/projects/ACMESharp.yml

```diff
@@ -1,8 +1,8 @@
+---
 name: ACMESharp
-desc: .NET Library and PowerShell module for the ACME protocol used by Let's Encrypt
+desc: ".NET Library and PowerShell module for the ACME protocol used by Let's Encrypt"
```

Comment:
I'm not 100% sure why a leading `.` here leads to such paranoid YAML generation,  but it seems like every string that starts with a `.` will be wrapped in quotes from now on.